tools/clang/test/CodeGenSPIRV/op.texture.mips-access.hlsl

```diff
@@ -0,0 +1,54 @@
+// Run: %dxc -T ps_6_0 -E main
+
+Texture1D        <float>  t1;
+Texture2D        <int2>   t2;
+Texture3D        <uint3>  t3;
+Texture1DArray   <float4> t4;
+Texture2DArray   <int3>   t5;
+
+// There is no .mips[][] for TextureCube      in HLSL reference.
+// There is no .mips[][] for TextureCubeArray in HLSL reference.
+
+// CHECK:  [[cu12:%\d+]] = OpConstantComposite %v2uint %uint_1 %uint_2
+// CHECK: [[cu123:%\d+]] = OpConstantComposite %v3uint %uint_1 %uint_2 %uint_3
+
+void main() {
+
+  uint pos = 7;
+  uint2 pos2 = uint2(1,2);
+  uint3 pos3 = uint3(1,2,3);
+
+// CHECK:        [[pos:%\d+]] = OpLoad %uint %pos
+// CHECK-NEXT:    [[t1:%\d+]] = OpLoad %type_1d_image %t1
+// CHECK-NEXT:    [[f1:%\d+]] = OpImageFetch %v4float [[t1]] [[pos]] Lod %uint_1
+// CHECK-NEXT:  [[val1:%\d+]] = OpCompositeExtract %float [[f1]] 0
+// CHECK-NEXT:                  OpStore %a1 [[val1]]
+  float  a1 = t1.mips[1][pos];
```

Comment:
Also change a few first indices to variable references and a few second indices to constant values?

This may require we evaluate the first index as rvalue constant.

Reply:
Done